docs/src/css/custom.css

```
@@ -75,6 +75,13 @@ div[class^='announcementBar_'] {
   font-weight: 500;
 }
 
+/* workaround for version switcher */
+div[class='navbar__items navbar__items--right'] {
+  li[class*='menu__list-item'] {
+    display: none;
+  }
+}
+
```

This feels jank, but I found no other way to solve the problem without completely rewriting `docs/src/components/version-switcher.tsx`.

---

I had some trouble selecting the right element while making this CSS and put the link within the div but I think this is better:

```suggestion
/* workaround for version switcher */
div[class*='navbar__items navbar__items--right'] li[class*='menu__list-item'] {
  display: none;
}

```